CMakeLists.txt

```diff
@@ -80,8 +81,8 @@ add_executable(threaded_io_test
   sourceFactoryGenerator.cc
   threaded_io_test.cc)
 
-target_include_directories(threaded_io_test PRIVATE "${PROJECT_BINARY_DIR}" ${HDF5_DIR}/include)
-target_link_directories(threaded_io_test PRIVATE ${HDF5_DIR}/lib)
+target_include_directories(threaded_io_test PRIVATE "${PROJECT_BINARY_DIR}" ${HDF5_DIR}/include ${MPI_DIR}/include)
```

Comment:
Does this change require MPI to be available even for the non-MPI builds?

Reply:
It does not. I will take that out. 
It was because the HDF5 version I am using in my testing is MPI-based.